lib/rules/no-test-support-import.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @fileOverview no importing of test files and no exporting in test files
+ */
+
+'use strict';
+
+const NO_IMPORT_MESSAGE =
+  'Do not import a file from test-support into production code, only into files located in test.';
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'disallow importing of "test-support" files in production code.',
+      category: 'Testing',
+      recommended: true,
+      url:
+        'https://github.com/ember-cli/eslint-plugin-ember/tree/master/docs/rules/no-test-support-import.md',
+    },
+    schema: [],
+    importMessage: NO_IMPORT_MESSAGE,
```

Comment:
For consistency sake, can we follow the pattern for error messages that [other](https://github.com/ember-cli/eslint-plugin-ember/blob/df59c8d7467831fd8fcd9f70bef71151c30ba8f7/lib/rules/classic-decorator-hooks.js#L9-L11) [rules](https://github.com/ember-cli/eslint-plugin-ember/blob/master/lib/rules/closure-actions.js#L25) do.

Reply:
updated to follow this pattern.